tests/integration/distributed-replicas/test_distributed_replicas.py

```diff
@@ -66,8 +66,9 @@ def _external_deployment_args_docker(num_shards, port=None):
     return set_deployment_parser().parse_args(args)
 
 
-@pytest.mark.parametrize('hosts', ['localhost,localhost', 'localhost'])
-def test_distributed_replicas(input_docs, hosts):
+@pytest.mark.parametrize('hosts', ['localhost,localhost', ['localhost','localhost'], 'localhost'])
+@pytest.mark.parametrize('ports', [lambda port1,port2: f'{port1},{port2}', lambda port1,port2: [port1,port2]])
```

Comment:
there is no need to use lambda here

Reply:
is there another way instead of lambda? `port1` and `port2` are generated in the test function